File: 브론즈 2 (5문제)/2355_시그마.c
// 참고 링크 : https://moon-gd.tistory.com/30
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

// 두 정수를 바꿔주는 함수
void swap(long long* a, long long* b);

int main() {
	long long a, b;
	scanf("%lld %lld", &a, &b);

	// a가 더 큰 입력으로 들어오는 경우, a와 b를 바꾸어준다.
	if (a > b) {
		swap(&a, &b);
	}

	printf("%lld", (a + b) * (b - a + 1) / 2);

	return 0;
}

// 두 정수를 바꿔주는 함수
void swap(long long* a, long long* b) {
	long long temp = *a;
	*a = *b;
	*b = temp;
}

File: 브론즈 2 (5문제)/2355_시그마.py
# 참고 링크 : https://moon-gd.tistory.com/30
A, B = map(int, input().split())

# A가 B보다 큰 경우, 서로 바꿔준다
if A > B:
    A, B = B, A

# 결과 출력
print((A + B) * (B - A + 1) // 2)
